src/client/pages/preferences/index.vue

```diff
@@ -70,6 +70,12 @@
 			<mk-switch v-model="enableInfiniteScroll">{{ $t('enableInfiniteScroll') }}</mk-switch>
 			<mk-switch v-model="disablePagesScript">{{ $t('disablePagesScript') }}</mk-switch>
 		</div>
+		<div class="_content">
+			<mk-select v-model="timestampFormat">
+				<template #label>{{ $t('timestampFormat') }}</template>
+				<option v-for="tf in timestampFormats" :value="tf" :key="tf">{{ tf }}</option>
```

Comment:
```suggestion
				<option v-for="tf in timestampFormats" :value="tf" :key="tf">{{ $t('_timestampFormats.' + tf) }}</option>
```

みたいにして

```yaml
_timestampFormats:
  relative: Relative
  absolute: Absolute
  detail: Detail
```

みたいなのを locales に追加したら多言語化できないかな？

Reply:
ありがとです 🙇‍♂️  。できました。